src/image_processing/inference/detection/hand_detection.py

```diff
@@ -57,34 +57,5 @@ def predict(self, img, should_draw_results):
 
         return boxes, img_resize, image_resize_drawn
 
-    def get_hand_from_img(self, image, boxes, enlargebox_px):
-        hands = []
-
-        for box in boxes:
-            x = box[0]
-            y = box[1]
-            w = box[2]
-            h = box[3]
-            # x, y, w, h = box
-            # enlarge box a bit
-            x -= enlargebox_px
-            y -= enlargebox_px
-            w += enlargebox_px * 2
-            h += enlargebox_px * 2
-
-            bottom_right_x = x + w
-            bottom_right_y = y + h
-
-            if x < 0:
-                x = 0
-            if y < 0:
-                y = 0
-            if bottom_right_x > image.shape[1]:
-                bottom_right_x = image.shape[1]
-            if bottom_right_y > image.shape[0]:
-                bottom_right_y = image.shape[0]
-
-            hand = image[y:bottom_right_y, x:bottom_right_x]
-            hands.append(hand)
-
-        return hands
```

Comment:
I think it's so common method that there's no point having it as a class method. Look that you have to pass the image to this function, as the class doesn't store the image, and you have to store boxes as well. 

And if it is a class method, then if you would like to get a hand from the image in some later stage, then the boxes or image could be outdated.

Having it as a function is a better way, because you can use this anytime, on any image and with any boxes that you want. You can crop whatever you want with this method. 

If you necessarily want this to be a class method, we can create a wrapper in a class.

Reply:
I don't :D It was just a suggestion.